packages/spanner/src/session-pool.js

```diff
@@ -35,18 +35,18 @@ var util = require('util');
  *
  * @param {module:spanner/database} database - The DB instance.
  * @param {object=} options - Configuration options.
+ * @param {number} options.acquireTimeout - Time in milliseconds before giving
+ *     up trying to acquire a session.
  * @param {boolean} options.fail - If set to true, an error will be thrown when
  *     there are no available sessions for a request. (Default: `false`)
  * @param {number} options.max - Maximum number of resources to create at any
- *     given time. (Default: `1`)
+ *     given time. (Default: Infinity)
```

Comment:
I think the change you're looking for is here - https://github.com/callmehiphop/gcloud-node/blob/4e63fbf477716b45794e8669ba1c0ca6c21ee567/packages/spanner/src/session-pool.js#L149